<launch>
  <node name="fake_localization" pkg="fake_localization" type="fake_localization" output="screen">
    <remap from="base_pose_ground_truth" to="/odom"/>
    <param name="odom_frame_id" value="odom"/>
    <param name="global_frame_id" value="ar_global"/>
    <param name="base_frame_id" value="base_footprint"/>
    <param name="delta_x" value="-1.0"/>
    <param name="delta_y" value="0.0"/>
    <param name="delta_yaw" value="3.14"/>
  </node>
</launch>
